packages/modules/viewer-react/src/types.ts

```diff
@@ -134,20 +141,59 @@ export interface ViewerInitializerParams extends ViewerIModelAppOptions {
 
 export interface ConnectedViewerProps {
   iTwinId: string;
-  iModelId: string;
+  iModelId?: string;
```

Comment:
i think we should leave this as required, as we need both itwin and imodel ids to open a remote connection model

Reply:
Done.